Nure/PullRequest/Bitbucket/BitbucketClient.cs

```diff
@@ -44,22 +44,13 @@ public BitbucketUser GetCurrentUser()
         {
             s_Logger.Info("Getting current user from Bitbucket.");
 
-            return new Url(m_BaseUrl).AppendPathSegments(API_VERSION, "user")
-                .WithBasicAuth(m_Username, m_Password)
-                .WithClient(m_FlurlClient)
-                .GetJsonAsync<BitbucketUser>()
-                .Result;
+            return new Url(m_BaseUrl).AppendPathSegments(API_VERSION, "user").WithBasicAuth(m_Username, m_Password).WithClient(m_FlurlClient).GetJsonAsync<BitbucketUser>().Result;
         }
 
         public List<BitbucketUser> GetDefaultReviewers()
         {
             s_Logger.Info($"Getting default reviewers of repository {m_RepositoryId} from Bitbucket.");
-            BitbucketPagedResult<BitbucketUser> reviewers = new Url(m_BaseUrl)
-                .AppendPathSegments(API_VERSION, "repositories", m_WorkspaceId, m_RepositoryId, "default-reviewers")
```

Comment:
Very nit picky, but is there a way  to parametrize the  URL building?
Some repeat and some dont  e.g. `.WithBasicAuth(m_Username, m_Password)
                .WithClient(m_FlurlClient)`